src/main.rs

```diff
@@ -78,36 +79,54 @@ async fn main() -> Result<()> {
                     None => PullDestination::MainStore,
                 };
                 let (sources, docker_config) = remote_server_options(matches)?;
-                pull::pull(uri, docker_config, sources, destination).await?;
+                let (key_files, annotations) = verification_options(matches)?;
+
+                // verify policy prior to pulling if keys listed, and keep the
+                // verified manifest digest of last iteration, even if all are
+                // the same:
+                let mut verified_manifest_digest: String = String::new();
```

Comment:
A better way is to use an `Option`:

```suggestion
let mut verified_manifest_digest: Option<String> = None;
```

Reply:
Indeed, thanks! Changed it in the last fixup commits; I can rebase after the review (I wonder what is preferable, to re-review it all, or for me to push fixup commits that will dissapear with a rebase after the review).